code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -114,6 +114,57 @@
 /obj/item/ammo_casing/magnum/scrap/prespawned
 	amount = 6
 
+//// .45 ACP ////
+
+/obj/item/ammo_casing/pistol_45
+	desc = "A .45 ACP bullet casing."
+	icon_state = "pistol_c_l"
+	spent_icon = "pistol_c_l-spent"
+	caliber = CAL_PISTOL
```

Comment:
Swap all these to CAL_45 to fix the pistole bug

Reply:
Should be fixed.